lib/OpenQA/WebAPI/Controller/Admin/Needle.pm

```diff
@@ -65,14 +65,10 @@ sub ajax {
     while (1) {
         my $column_index = $self->param("order[$index][column]");
         my $column_order = $self->param("order[$index][dir]");
-        if (
-            !(
-                   defined($column_index)
-                && $column_index < scalar(@columns)
-                && ($column_order eq 'asc' || $column_order eq 'desc')))
-        {
-            last;
-        }
+        last
```

Comment:
I always tell the interns to avoid post conditions if it leads to line breaks. and here you do the other way around. And I somehow don't find either before or after very readable ;(